catalog/openshift-cert-manager-operator/bundle-v1.14.0.yaml

```diff
@@ -347,7 +347,7 @@ relatedImages:
   name: cert-manager-operator-rhel9-2df1899f8c9f877cb9e1239b860f34c0e7a27d72d87b561fa79a5d7319699d59-annotation
 - image: registry.redhat.io/cert-manager/cert-manager-operator-rhel9@sha256:2df1899f8c9f877cb9e1239b860f34c0e7a27d72d87b561fa79a5d7319699d59
   name: cert-manager-operator
-- image: registry.redhat.io/cert-manager/jetstack-cert-manager-acmesolver-rhel9@sha256:9f185dca969322dae0806b3f201fb2fdb43924a70eaa16e2d543355da0b4efbe
+- image: registry.redhat.io/cert-manager/jetstack-cert-manager-acmesolver-rhel9@sha256:fe6e8d777ea617eb1de710665d4c989eba56d0724a74541b96db21116b916027
```

Comment:
Is this supposed to be same in both v1.14.0 and v1.14.1 catalog bundle?

Reply:
Same as https://github.com/openshift/cert-manager-operator-release/pull/79#discussion_r1879574775